BlueShift-iOS-Extension-SDK/BlueShiftPushNotification.m

```diff
@@ -92,9 +98,11 @@ - (NSArray *)carouselAttachmentsDownload:(UNNotificationRequest *)request {
                  NSString  *filePathToWrite = [NSString stringWithFormat:@"%@/%@", documentsDirectory, attachmentName];
                  [imageData writeToFile:filePathToWrite atomically:YES];
                  
-                 NSError *error3;
-                 UNNotificationAttachment *attachment = [UNNotificationAttachment attachmentWithIdentifier:attachmentName URL:URL options:nil error:&error3];
-                 NSLog(@"%@", error3);
+                 NSError *error;
+                 UNNotificationAttachment *attachment = [UNNotificationAttachment attachmentWithIdentifier:attachmentName URL:URL options:nil error:&error];
+                 if (error) {
+                     NSLog(@"[Blueshift] Failed to create carousel image attachment %@", error);
```

Comment:
should we add a check for the debug mode here as well?

Reply:
if we add debug in extension SDK, we won't be able to see these logs in the prod version of app. The Blueshiftlog functinality is built only for core SDK.

Reply:
ok, there is one debug flagged log above. hence got confused.

Reply:
Thanks for picking this up, I will remove the #debug condition of missing api key log.